### ISSUE TITLE
"Multiply-claimed blocks" error when running e2fsck on SD card

### ISSUE THREAD

**Issue by DanMan** [ValveSoftware/SteamOS#1103]
I've mounted the SD card of my Steam Deck on my desktop Fedora 38 system and it put its file system in read-only mode because it had all kinds of errors:

* Directory block failed checksum
* bad block bitmap checksum

Running `e2fsck -D` on it then resulted in plenty of the errors in the title.

Is this a known issue (false alert?) because you're doing weird things with the files in Proton? I would have never found out, if I hadn't mounted the card on my main PC and had a look at `dmesg`. I've only ever used that card with the Steam Deck and it was ext4 formatted through it, too.

I doubt it's a faulty card, because it's a brand new Samsung one, but it could be, of course.

**Comment by bertogg** [comment]
I don't think that Proton has anything to do with this. Did you test the card with `badblocks` ?
